frost-ed25519/Cargo.toml

```diff
@@ -52,6 +52,8 @@ serialization = ["serde", "frost-core/serialization"]
 ## can use `serde` to serialize structs with any encoder that supports
 ## `serde` (e.g. JSON with `serde_json`).
 serde = ["frost-core/serde"]
+## Enable cheater detection
+cheater-detection = ["frost-core/cheater-detection"]
```

Comment:
We will also need to enable it by default in the ciphersuite crates

Reply:
Done in [4d26417](https://github.com/ZcashFoundation/frost/pull/564/commits/4d26417d1d039f2fd9ba38b3b7f149cbd8a7e297)